packages/relay-compiler/generic/codegen/RelayCodegenRunner.js

```diff
@@ -165,7 +168,7 @@ class RelayCodegenRunner {
         const query = {
           expression: [
             'allof',
-            config.watchmanExpression,
+            RelayCodegenWatcher.buildWatchExpression(config),
```

Comment:
I think this change here won't be enough - if watchman is disabled, then calling `getDirtyWriters` will fail here.